apps/sms/js/navigation.js

```diff
@@ -152,7 +171,7 @@ var Navigation = {
     //   - Panel is still transitioning
     //   - trying to navigate when the app is not loaded completely
     var notReadyNavigate = (panel !== this.defaultPanel && !this.isReady);
-    if (this.transitioning || notReadyNavigate) {
+    if (this.transitionPromise || notReadyNavigate) {
       queuedPanel = {
         panel: panel,
         args: args
```

Comment:
I've been thinking that we could likely not nullify `this.transitionPromise` and encapsulate everything here with this.transitionPromise.then(...). But this is more than we'd want to do here obviously :)